packages/cicero-server/package.json

```diff
@@ -46,10 +46,10 @@
     "chai-as-promised": "7.1.1",
     "chai-things": "0.2.0",
     "decache": "4.4.0",
-    "eslint": "8.2.0",
+    "eslint": "^7.32.0",
```

Comment:
Did you mean to downgrade `eslint`?

Reply:
You are right, it was a mistake. Corrected at https://github.com/accordproject/cicero/pull/746/commits/e6d4cd58397030a9e7511cde0e6aa6898dec0d36.